cpp/include/cudf/strings/char_types/char_types_enum.hpp

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
```

Comment:
Since this is old code, maybe the copyright year should remain as when it was created?